content/en/docs/File Format/implementationstatus.md

```diff
@@ -0,0 +1,178 @@
+
+### Other format level features
+
++-------------------------------------------+-------+--------+--------+-------+-------+
+|                                           | C++   | Python | Java   | Go    | Rust  |
+|                                           |       |        |        |       |       |
++===========================================+=======+========+========+=======+=======+
+| xxHash Bloom filters                      |       |        |        |       |       |
++-------------------------------------------+-------+--------+--------+-------+-------+
+| bloom filter length                       |       |        |        |       |       |
++-------------------------------------------+-------+--------+--------+-------+-------+
+| Statistics min_value, max_value           |       |        |        |       |       |
++-------------------------------------------+-------+--------+--------+-------+-------+
+| Column index                              |       |        |        |       |       |
++-------------------------------------------+-------+--------+--------+-------+-------+
+| Offset index                              |       |        |        |       |       |
++-------------------------------------------+-------+--------+--------+-------+-------+
+| Modular encryption                        |       |        |        |       |       |
++-------------------------------------------+-------+--------+--------+-------+-------+
+| Page CRC32 checksum                       |       |        |        |       |       |
++-------------------------------------------+-------+--------+--------+-------+-------+
+| Modular encryption                        |       |        |        |       |       |
++-------------------------------------------+-------+--------+--------+-------+-------+
+
+High level data API-s for parquet feature usage
+===============================================
+
++----------------------------------------------+-------+--------+--------+-------+-------+
+| Format                                       | C++   | Python | Java   | Go    | Rust  |
+|                                              |       |        |        |       |       |
++==============================================+=======+========+========+=======+=======+
+| Hive-style partitioning                      |       |        |        |       |       |
++----------------------------------------------+-------+--------+--------+-------+-------+
+| Partition pruning on the partition column    |       |        |        |       |       |
++----------------------------------------------+-------+--------+--------+-------+-------+
+| External column data                         |       |        |        |       |       |
++----------------------------------------------+-------+--------+--------+-------+-------+
+| RowGroup Sorting column                      |       |        |        |       |       |
```

Comment:
Also, what does this mean?

Reply:
https://github.com/apache/parquet-format/blob/1603152f8991809e8ad29659dffa224b4284f31b/src/main/thrift/parquet.thrift#L834